financialaid/views_test.py

```diff
@@ -243,17 +243,28 @@ def test_context(self):
             GA_TRACKING_ID=ga_tracking_id,
             REACT_GA_DEBUG=react_ga_debug,
             EDXORG_BASE_URL=base_url,
-        ):
+        ), patch('ui.templatetags.render_bundle._get_bundle') as get_bundle:
             response = self.client.get(self.review_url)
-            assert response.context['has_zendesk_widget'] is True
-            assert response.context['is_public'] is True
-            self.assertContains(response, 'Share this page')
-            assert json.loads(response.context['js_settings_json']) == {
-                'gaTrackingID': ga_tracking_id,
-                'reactGaDebug': react_ga_debug,
-                'authenticated': True,
-                'edx_base_url': base_url,
-            }
+
+        bundles = [bundle[0][1] for bundle in get_bundle.call_args_list]
```

Comment:
Not really, I unindented some code but I reverted it for the other blocks because I wanted to minimize the change. I'll revert it here too